### PR TITLE
Modification de texte

### PR DESCRIPTION
**Pull request by CuauhFrMx**
## Pull request checklist

Please check if your PR fulfills the following requirements:
- [x] You can **build successfully** the whole project with your changes locally
- [x] You use the coding conventions and the **naming conventions** of the project
- [x] You use **tabs** for indentation
- [x] You make sure that your PR is **not a duplicate**
- [x] This PR is **ready** to be reviewed and merged
- [x] This PR merges into the **`development`** branch
- [x] There is no **`TODO`** in the code
- [x] There is no spelling or grammatical errors in the code
- [x] Details of the issue or feature are documented below
- [x] This PR is **not** a **breaking change** (e.g. changes that would cause existing functionality to change)

## Proposed changelog

| Français `fr` |
| --- |
| Uniquement des modifications de texte, et aussi changement de **1 jour(s)** par **1 jour** dans la homepage |

### CODE REVIEW

**Review by sivelswhy — APPROVED** [review]
c'est carré

**Comment by CuauhFrMx** [comment]
> c'est carré

merci :)

**Comment by moeenio** [comment]
Merci!

File: src/views/HomepageView.vue [hunk 1]
```diff
@@ -620,7 +621,7 @@
 						</div>
 						<ion-label>
 							<h2>Pas de devoirs</h2>
-							<p>Vous n'avez aucun devoir à faire durant 7 jours.</p>
+							<p>Vous n'avez aucun travail à faire dans les 7 prochains jours.</p>
```
Comment (moeenio) [inline src/views/HomepageView.vue:624]: _pour_ les 7 prochains jours plutôt?

**Review by moeenio — APPROVED** [review]

**Pull request merged**